lib/better_errors.rb

```diff
@@ -139,6 +144,7 @@ def self.default_editor
   require "binding_of_caller"
   require "better_errors/exception_extension"
   BetterErrors.binding_of_caller_available = true
+  BetterErrors.maximum_variable_inspect_size ||= 100_000
```

Comment:
If `binding_of_caller` is not available (raises `LoadError`), this line won’t be evaluated. I’m not sure where the best location for setting default values would be. Perhaps inside the class definition? Perhaps right after this block?